### PR TITLE
Remove jQuery `.attr` from the label edit exclusive checkbox

### PR DESCRIPTION
**Pull request by yardenshoham**
- Switched from jQuery `attr` to plain javascript `getAttribute`
- Tested the label edit exclusive checkbox and it works as before

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

File: web_src/js/features/comp/LabelEdit.js [hunk 1]
```diff
@@ -6,23 +6,23 @@ function isExclusiveScopeName(name) {
...
 
-  if (isExclusiveScopeName($nameInput.val())) {
-    $exclusiveField.removeClass('muted');
-    $exclusiveField.removeAttr('aria-disabled');
-    if ($exclusiveCheckbox[0].checked && $exclusiveCheckbox.data('exclusive-warn')) {
-      $exclusiveWarning.removeClass('tw-hidden');
+  if (isExclusiveScopeName(nameInput.value)) {
+    exclusiveField?.classList.remove('muted');
+    exclusiveField?.removeAttribute('aria-disabled');
+    if (exclusiveCheckbox.checked && exclusiveCheckbox.getAttribute('data-exclusive-warn')) {
```
Comment (wxiaoguang) [inline web_src/js/features/comp/LabelEdit.js:17]: Here is a regression bug, the code below uses `$exclusiveCheckbox.data('exclusive-warn', ...` and can't be read by `getAttribute`

Comment (silverwind) [reply web_src/js/features/comp/LabelEdit.js:17]: `.data("foo")` is equivalent to `.getAttribute("data-foo")` for plain string data, no?

Comment (wxiaoguang) [reply web_src/js/features/comp/LabelEdit.js:17]: `getAttribute('data-foo')` can't read `$().data('foo', 'value')`, see below `$exclusiveCheckbox.data('exclusive-warn',...)`

Comment (silverwind) [reply web_src/js/features/comp/LabelEdit.js:17]: Ah indeed. I forgot that `.data()` does not actually write HTML attributes. Should have refactored the whole file.

Comment (wxiaoguang) [reply web_src/js/features/comp/LabelEdit.js:17]: ->  Refactor LabelEdit #32752

**Pull request merged**